scala/ppml/src/main/scala/com/intel/analytics/bigdl/ppml/vfl/FLModel.scala

```diff
@@ -19,21 +19,25 @@ package com.intel.analytics.bigdl.ppml.vfl
 import com.intel.analytics.bigdl.DataSet
 import com.intel.analytics.bigdl.dllib.feature.dataset.MiniBatch
 import com.intel.analytics.bigdl.ppml.FLClient
+import com.intel.analytics.bigdl.ppml.utils.DataFrameUtils
 import com.intel.analytics.bigdl.ppml.vfl.nn.VflNNEstimator
+import org.apache.spark.sql.DataFrame
 
 class FLModel() {
   val estimator: VflNNEstimator = null
-  def fit(trainData: DataSet[MiniBatch[Float]],
-          valData: DataSet[MiniBatch[Float]],
+  def fit(trainData: DataFrame,
+          valData: DataFrame,
```

Comment:
need to specify feature columns and (optionally) label columns, similar to Orca estimator

Reply:
The API should be similar to https://github.com/intel-analytics/BigDL/blob/branch-2.0/python/orca/src/bigdl/orca/learn/pytorch/pytorch_spark_estimator.py#L104